mpp/shmemx.h4

```diff
@@ -46,6 +42,9 @@ include(shmemx_c_func.h4)dnl
 /* C11 Generic Macros */
 #elif (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L && !defined(SHMEM_INTERNAL_INCLUDE))
 
+/* SHMEMX constant(s) are included in MAX_HINTS value in shmem-def.h */
+#define SHMEMX_MALLOC_NO_BARRIER (1l<<2)
```

Comment:
This needs to go inside the `extern "C"` section - this is why all the tests are failing.  Can you please also group the "SHMEMX" constants together (i.e., put this just below `SHMEMX_CTX_BOUNCE_BUFFER`)?

Reply:
Thank you for catching this! 👍